src/types/user.types.ts

```diff
@@ -3,7 +3,10 @@ import { Keywords } from './keywords.types';
 
 export enum RoleName {
   AppAdmin = 'App-admin',
-  Curator = 'Curator',
+  Curator = 'Curator', // TODO: remove
+  DoiCurator = 'Doi-Curator',
+  SupportCurator = 'Support-Curator',
+  PublishingCurator = 'File-Curator',
```

Comment:
Har avtalt renaming av denne
```suggestion
  PublishingCurator = 'Publishing-Curator',
```